tstest.ts

```diff
@@ -66,18 +66,30 @@ import Pulsar = require('./index');
     scope: "scope"
   });
 
-  const authToken: Pulsar.AuthenticationToken = new Pulsar.AuthenticationToken({
+  const authToken1: Pulsar.AuthenticationToken = new Pulsar.AuthenticationToken({
     token: 'foobar',
   });
 
+  const authToken2: Pulsar.AuthenticationToken = new Pulsar.AuthenticationToken({
```

Comment:
Seems authToken2 and authToken3 are not used. We can remove it

Reply:
You right, I keep the async one.